app/layout.tsx

```diff
@@ -2,7 +2,7 @@ import 'tailwindcss/tailwind.css';
 import './globals.css'
 import { Montserrat as Inter } from 'next/font/google'
 
-import Navbar from '@/components/navbar';
+import Navbar from '@/components/Navbar';
```

Comment:
the filename is `navbar` that's why CI failed

```
import Navbar from '@/components/navbar';
```

Reply:
Oye, ya varias veces le he cambiad de `navbar` a `Navbar` por consistencia, y como que no le gusta, sabes que pedo?